extensions/wikia/ImageReview/ImageReviewHelper.class.php

```diff
@@ -1,620 +0,0 @@
-<?php
-
-use \Wikia\Logger\WikiaLogger;
-
-/**
- * ImageReview Helper
- *
- * @author (contributing) Adam Karmiński <adamk@wikia-inc.com>
- */
-class ImageReviewHelper extends ImageReviewHelperBase {
```

Comment:
This is the main class which just did way too much. It's been broken apart into `ImageStateUpdater`, `ImageCountGetter`, `ImageListGetter`, `AbandonedWorkResetter`, `StatsDataGetter`, and `ImageReviewOrderGetter`